File: Prod.RutaDigital.Entidades/Modulo/Modulo.cs
namespace Prod.RutaDigital.Entidades;

public class Modulo
{
    public int id_modulo { get; set; }
    public int orden { get; set; }
    public string? codigo { get; set; }
    public string? nombre { get; set; }
    public string? descripcion { get; set; }
    public decimal peso { get; set; }
    public string? imagen { get; set; }
}

File: Prod.RutaDigital.Presentacion.Angular/ClientApp/src/app/autodiagnostico/interfaces/modulo.ts
import { Pregunta } from './pregunta';
import { NivelMadurez } from './nivel-madurez';
import { CapacitacionResultado } from 'src/app/recomendaciones/interfaces/capacitacion-resultado';

export interface Modulo {
  id_modulo?: number;
  orden?: number;
  codigo?: string;
  nombre?: string;
  descripcion?: string;
  peso?: string;
  imagen?: string;

  preguntas?: Pregunta[];

  niveles?: NivelMadurez[];

  capacitaciones?: CapacitacionResultado[];

  completado?: boolean;
}
